networkmanager/networkmanager.go

```diff
@@ -125,6 +125,11 @@ type bandwidthNetConf struct {
 	EgressBurst  uint64 `json:"egressBurst,omitempty"`
 }
 
+type firewallNetConf struct {
```

Comment:
please add  comment in commit why we need this plugin.

Reply:
Add description for commit